com.biglybt.BiglyBT.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=BiglyBT
```

Comment:
actuallly this file is in the tarball, you should use that one and not add it here.

Reply:
The command and icon are different

Reply:
they can be changed using `desktop-file-edit`.

Reply:
Alright!, updated it